us-02/us-02.py

```diff
@@ -38,6 +39,9 @@ def responseOption():
     #     print("Response Length = " + str(len(responseValue)))
     # else:
     #     print("Response Length = " + str(len(responseValue)))
+    
```

Comment:
i think this will work for now. we'll have to return the truncated response and/or the tokenized set back to the driver code.